engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -45,17 +45,20 @@ public static DataflowError withoutTrace(Object payload, Node location) {
    * @param trace a specific stack trace
    * @return a new dataflow error
    */
-  public static DataflowError withTrace(Object payload, Node location, StackTraceElement[] trace) {
-    var error = new DataflowError(payload, location);
-    error.setStackTrace(trace);
-    return error;
+  public static DataflowError withTrace(Object payload, AbstractTruffleException prototype) {
+    return new DataflowError(payload, prototype);
```

Comment:
You may want to update the Javadoc comment above as the arguments have changed.

Reply:
ayyy true